### PR TITLE
feat: GEOT-454: styles for adding posts

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/app-andrew/src/pages/group/styles.scss [hunk 1]
```diff
@@ -39,8 +39,13 @@
       position: sticky;
       top: 0;
 
+      > div {
+        display: none;
+      }
+
       .avatarNamedDesc {
         padding: 10px;
+        display: block;
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/group/styles.scss:48]: hidden just for now - I'm waiting for https://geotags.atlassian.net/browse/GEOT-434

File: packages/app-andrew/src/pages/group/styles.scss [hunk 2]
```diff
@@ -62,17 +67,21 @@
...
             margin-left: 10px;
+            display: none;
           }
         }
 
         .desc {
           margin-top: 5px;
           @include font-size(sm);
           @include font-weight(semiBold);
+          display: none;
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/group/styles.scss:78]: hidden just for now - I'm waiting for https://geotags.atlassian.net/browse/GEOT-434

File: packages/app-andrew/src/pages/group/styles.scss [hunk 3]
```diff
@@ -62,17 +67,21 @@
...
         }
       }
 
       .navmock {
+        display: none;
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/group/styles.scss:83]: hidden just for now - I'm waiting for https://geotags.atlassian.net/browse/GEOT-434

File: packages/app-andrew/src/pages/group/styles.scss [hunk 4]
```diff
@@ -107,6 +116,7 @@
         width: 100%;
         padding: 12px 14px;
         justify-content: flex-start;
+        display: none;
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/group/styles.scss:119]: hidden just for now - I'm waiting for https://geotags.atlassian.net/browse/GEOT-434

**Review by mariusz-kabala — APPROVED** [review]

**Pull request merged**